dev/phoenix-personality.sh

```diff
@@ -87,9 +87,11 @@ function personality_globals
 
   # Set docker container to run with 20g. Default is 4g in yetus.
   # See HBASE-19902 for how we arrived at 20g.
-  # TODO Doesn't seem to have effect in Yetus 0.12, set in cli instead
   #shellcheck disable=SC2034
-  DOCKERMEMLIMIT=20g
+  DOCKER_MEMORY=20g
+
+  #Allow some swaping to avoid OOM errors
+  DOCKER_EXTRAARGS+=( "--memory-swap=24g" )
```

Comment:
Is this a proven way?

Reply:
I was just looking up https://docs.docker.com/config/containers/resource_constraints/ and found this:
```
--memory-swap is a modifier flag that only has meaning if --memory is also set.
```

What we specify with `DOCKERMEMLIMIT` is treated as `--memory` by Yetus?

Reply:
> Is this a proven way? (Just asking in case any Hadoop project is following this and it's working fine for them)

Not that I know of. 
I was just looking at the docker memory management options, and this seemed like a solution without overcommitting the host memory.

Reply:
> What we specify with `DOCKERMEMLIMIT` is treated as `--memory` by Yetus?

Yes, DOCKER_MEMORY is passed as --memory= to docker by yetus.
(The DOCKERMEMLIMIT variable name was wrong, or at least obsolete)